README.md

```diff
@@ -4,128 +4,108 @@
+- [Install](#install)
+- [Dependencies](#dependencies)
+- [Usage](#usage)
+- [Contributing](#contributing)
+- [Community meetings](#community-meetings)
+- [Office Hours](#office-hours)
+
+## More information
 
 - [Documentation](https://docs.m3db.io/)
 - [Developer Guide: Getting Started](https://github.com/m3db/m3/blob/master/DEVELOPMENT.md)
 - [Slack](http://bit.ly/m3slack)
 - [Forum (Google Group)](https://groups.google.com/forum/#!forum/m3db)
-- [Twitter](https://twitter.com/m3metrics)
-
-## Community meetings
-
-M3 contributors and maintainers have monthly (every four weeks) meetings. Join our M3 meetup group to receive notifications on upcoming meetings: 
-[https://www.meetup.com/M3-Community/](https://www.meetup.com/M3-Community/).
 
-Recordings of past meetups can be found here: [https://vimeo.com/user/120001164/folder/2290331](https://vimeo.com/user/120001164/folder/2290331).
+## Install 
 
-## Office Hours
 
-Members of the M3 team will hold office hours on the third Thursday of every month from 11-1pm EST. In order to join, make sure to sign up for a slot here: [https://calendly.com/chronosphere-intro/m3-community-office-hours](https://calendly.com/chronosphere-intro/m3-community-office-hours).
 
-## Test it out
+### Dependencies 
 
-The easiest way to testing out M3 is to follow one of the guides from the documentation. For a fully comprehensive getting started guide, see our [single node how-to](https://docs.m3db.io/how_to/single_node/).
-
-### Starting a node
+The simplest and quickest way to try M3 is to use Docker, read [the M3 quickstart section](https://m3db.io/docs/quickstart) for other options. 
 
 ```
-# to build a local m3dbnode process
-make m3dbnode (note that we currently require at least Go 1.10 or higher)
 
-# run it with the sample configuration
-./bin/m3dbnode -f ./src/dbnode/config/m3dbnode-local-etcd.yml
-```
+## Usage 
 
-To cross-compile and build for Linux AMD64 build with `make m3dbnode-linux-amd64`.
-
-### Creating a namespace to store metrics
+The below is a simplified version of the [M3 quickstart guide](https://m3db.io/docs/quickstart/docker/), and we suggest you read that for more details.
 
+1. Start a container
+```shell
+docker run -p 7201:7201 -p 7203:7203 --name m3db -v $(pwd)/m3db_data:/var/lib/m3db quay.io/m3db/m3dbnode:v1.0.0
 ```
+
+2. Create a placement and namespace
+```shell
+#!/bin/bash
 curl -X POST http://localhost:7201/api/v1/database/create -d '{
   "type": "local",
   "namespaceName": "default",
-  "retentionTime": "2h"
-}'
+  "retentionTime": "12h"
+}' | jq .
 ```
 
-### Test RPC
-
-To test out some of the functionality of M3DB there are some user friendly HTTP JSON APIs that you can use. These use the DB node cluster service endpoints.
-
-Note: performance sensitive users are expected to use the more performant endpoints via either the Go `src/dbnode/client/Session` API, or the GRPC endpoints exposed via `src/coordinator`.
-
-#### Write a datapoint
-
+3. Ready a namespace
```

Comment:
Maybe mark the namespace as ready?

Reply:
This is what it's called in the quickstarts right now, we could change it everywhere later I guess